### PR TITLE
[cli] Update `project` command to new structure

### PR DESCRIPTION
**Pull request by trek**
Before:
CleanShot 2023-08-31 at 13 29 29@2x

After:
CleanShot 2023-08-31 at 13 31 50@2x

### CODE REVIEW

**Review by cb1kenobi — APPROVED** [review]
Wow!

File: packages/cli/src/commands/project/command.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      options: [],
+      examples: [],
+    },
+  ],
+  options: [
+    {
+      name: 'next',
+      description: 'Show next page of results',
+      argument: 'MS',
+      shorthand: 'n',
```
Comment (EndangeredMassa) [inline packages/cli/src/commands/project/command.ts:51]: **suggestion (blocking):** ‏use capital

```suggestion
      shorthand: 'N',
```

@trek The shorthand for this option is capital.

https://github.com/vercel/vercel/blob/e43191b1866da70a3dab3815a3f2176942240ef3/packages/cli/src/commands/project/index.ts#L25

**Comment by EndangeredMassa** [comment]
@trek test comment

File: packages/cli/src/commands/project/command.ts (hunk 1 above)
Comment (trek) [reply packages/cli/src/commands/project/command.ts:51]: https://github.com/vercel/vercel/pull/10460

**Pull request merged**